.gitignore

```diff
@@ -1,4 +1,5 @@
-.vscode
+.vscode/*
```

Comment:
I'm wondering why `.vscode` is excluded since it already contains a launch config.